dags/ods/survey_cake/dags/questionnaire_2_bigquery.py

```diff
@@ -27,7 +29,8 @@
 )
 with dag:
     if bool(os.getenv("AIRFLOW_TEST_MODE")):
-        FILENAMES: Dict[str, Dict] = {"fixtures/data_questionnaire.csv": {}}
+        filepath = Path(AIRFLOW_HOME) / "dags/fixtures/data_questionnaire.csv"
+        FILENAMES: Dict[str, Dict] = {str(filepath): {}}
```

Comment:
nitpick: we could use `from __future__ import annotations` and replace it as the following

```suggestion
        FILENAMES: dict[str, dict] = {str(filepath): {}}
```

Reply:
thanks your suggestion.